### PR TITLE
Ft: Github workflow and tests added

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Added manageTokens tests (to check if the user provided token ID is valid or not, and throw necessary errors)
- Added github workflow of build, tests and lint
- Major changes is regarding the yarn.lock file
- Added manageAdmin tests (to check the status of wallet account ID provided, add/remove it from admin list)

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
After we will merge `ft-format` into this branch the conflicts will be resolved
And even the tests will pass after that (since vercel has some prettier config requirement)
So after that we can merge this into `develop

File: .eslintrc.js [hunk 1]
```diff
@@ -42,5 +42,6 @@ module.exports = {
   globals: {
     process: 'readonly',
     Buffer: true,
+    global: true,
```
Comment (robert-zaremba) [inline .eslintrc.js]: Let's enable global only in test files. Could you update the config?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -31,9 +30,39 @@ jobs:
...
+      - name: Checkout the code
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Check Yarn version
+        run: yarn --version
+      - name: Install dependencies
+        run: yarn install
```
Comment (robert-zaremba) [inline .github/workflows/main.yml]:
```suggestion
```

Comment (robert-zaremba) [reply .github/workflows/main.yml]: Let's run lint and format in the same job. No need to install deps twice. Test should run in parallel though.

**Review by robert-zaremba — APPROVED** [review]
pre-approving

File: .eslintrc.js (hunk 1 above)
Comment (Megha-Dev-19) [reply .eslintrc.js]: okay

File: .github/workflows/main.yml (hunk 2 above)
Comment (Megha-Dev-19) [reply .github/workflows/main.yml]: okay

**Pull request merged**